.github/workflows/rust.yml

```diff
@@ -151,7 +151,9 @@ jobs:
 
       - name: Deploy interstellar
         if: github.ref == 'refs/heads/interstellar'
-        run: bash scripts/deploy_docker.sh interstellar
+        run: |
+          bash scripts/deploy_docker.sh interstellar
+          curl -H "Content-Type: application/json" -X POST -d '{"push_data": {"tag": "staging" }}' ${{ AUTODEPLOY_URL }}
```

Comment:
is `staging` the correct tag here?

Reply:
shouldn't this be `interstellar`?

Reply:
wasn't this supposed to be `staging` for the deployment prefix?